Platform/obc/hal/comms_hal.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#ifndef COMMS_HAL_H
+#define COMMS_HAL_H
+
+#include <inttypes.h>
+#include <csp/csp.h>
+#include "services.h"
+#include "communication_service.h"
+
+/* Should optimize this*/
+#ifdef SBAND_IS_STUBBED
+  typedef enum{
+    FUNC_PASS  = 0,
+    BAD_READ   = 1,
+    BAD_WRITE  = 1,
+    BAD_PARAM  = 2,
+    IS_STUBBED = 0,
+  } STX_return;
```

Comment:
nice. We probably want this to be included even if `SBAND_IS_STUBBED` is not defined, right?

Reply:
Well, if it's not defined, the code probably won't return IS_STUBBED at all. I just added this because the functions had to return something and I added it here in case we want to change 0 to something else for test purposes.

Reply:
true. If the macro isn't defined, then we won't have access to this enum at all.
OH I see, there's another definition in `sTransmitter.h` - I see what you're doing.. Ignore me :)